### DISCUSSION TITLE
viscous diffusion in PeleC

### DISCUSSION THREAD

**Discussion by jagmohas** [AMReX-Combustion/PeleC#614] [Q&A] [answered]
I am trying to understand this calculation from GradUtil.H in PeleC. Could you point me to a source where I can understand this calculation? I don't understand why we add terms for j+1 but subtract terms for j-1. 
` td(i, j, k, 0) = 0.25 * dx1inv *
                     (q(i, j + 1, k, QU) + q(i - 1, j + 1, k, QU) -
                      q(i, j - 1, k, QU) - q(i - 1, j - 1, k, QU)); // dudy`
   ` td(i, j, k, 1) = 0.25 * dx1inv *
                     (q(i, j + 1, k, QV) + q(i - 1, j + 1, k, QV) -
                      q(i, j - 1, k, QV) - q(i - 1, j - 1, k, QV)); // dvdy`

Thanks.

**Comment by drummerdoc** [comment, marked as answer]
![Screen Shot 2023-04-03 at 3 32 53 PM](https://user-images.githubusercontent.com/1464103/229632397-8d4e44c5-c0bd-4c34-9f3a-05631673af92.png)

Does this help?

**Reply by jagmohas** [reply]
Thanks Marc, I sketched something similar, but when comparing it with the code I missed that we don't have a bracket for qc+qd. Thanks a ton.
